.github/workflows/ci-test.yml

```diff
@@ -1,7 +1,4 @@
 on:
-  push:
-    branches:
-      - '*'
```

Comment:
@Vizualni Without this it's just running CI twice against every PR. If you want CI running on all branches I would turn it off for PRs.